scripts/dev_cli.sh

```diff
@@ -328,7 +346,7 @@ cmd_tests() {
 	       --env CFLAGS="$cflags" \
 	       --env TARGET_CC="$target_cc" \
 	       "$CTR_IMAGE" \
-	       ./scripts/run_unit_tests.sh "$@" || fix_dir_perms $? || exit $?
+	       ./scripts/run_unit_tests.sh "${saved_args[@]}" || fix_dir_perms $? || exit $?
```

Comment:
Sorry I should've been clearer.

Originally `$@` is only what is left after `--`. Now you're passing everything, including parameters which are supposed to be only for `dev_cli.sh`, to all the test scripts.

That does not cause problems at the moment, but that isn't correct conceptually.

Instead of introducing `saved_args`, you can append `--hypervisor` to `$@` after the `while` loop which parses parameters. I think that will also make your patch shorter.